package.json

```diff
@@ -8,8 +8,8 @@
     "predump": "npm run -s clean",
     "dump": "npm run -s dump:galen:local",
     "test": "concurrently -p name -n lint:js,lint:wc,polymer:test \"npm run lint:js\" \"npm run lint:wc\" \"npm run test:galen:local\" \"polymer test --skip-plugin sauce\"",
```

Comment:
Hmm, there's still `lint:js` references here, but the script won't know to update those.  Maybe we should just leave that as is and add the `lint:js` line update to the patch?

Reply:
Or, better yet, we could update the script to look for references of `lint:html` and change those to `lint:js`

Reply:
Or, just replace with general `lint` like your last commit 😛

Reply:
I just changed the references to `npm run lint` instead.